presto-parquet/src/main/java/com/facebook/presto/parquet/ParquetTypeUtils.java

```diff
@@ -256,17 +259,31 @@ public static boolean isValueNull(boolean required, int definitionLevel, int max
     // copied from presto-hive DecimalUtils
```

Comment:
is this still true?

Reply:
Good catch, thank you @elharo I've deleted it.